SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+We aim to keep Timescale safe for everyone. 
+Publicly disclosing bugs in a public forum puts everyone in the Timescale community at risk.
+
+## Supported Versions
+
+Our users should always upgrade to the latest version.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.6.x   | :white_check_mark: |
+| 1.x   | :x:                |
```

Comment:
```suggestion

Please take a look at our [Support Policy](https://www.timescale.com/legal/support-policy).
```

@clarkbw here.

Reply:
> https://www.timescale.com/legal/support-policy

I don't think this page helps anyone with the question of "is this release where I found an issue a supported version?". I didn't want us to have this page refer to `2.6.x` when we're on to `3.1.0` because it could create confusion.  I was hoping for some page in the repo with the latest supported version list.  Or perhaps we could say the latest major version published in the repo is supported. Not sure.

Reply:
This was the original suggestion.

Reply:
The original suggestion was to include the supported versions in this page but that creates a place where we need to ensure we have updated the supported version as new versions are released. If we don't have a place where we declare supported versions I'm not sure what we should do next, but having a page like that might be an approach.